### ISSUE TITLE
Problem adding port to OPENCART_HOST env variable

### ISSUE THREAD

**Issue by mattia85** [bitnami/bitnami-docker-opencart#75]
**Unable to run opencart container using different port**

I'm setting env variable **OPENCART_HOST** to **localhost:8084** and **ports: 8084:80**, but all occurrences of the host in the html source code are replaced with the OPENCART_HOST value surrounded by square brackets, like the following link.
`<link href="http://[localhost:8084]/image/catalog/cart.png" rel="icon" />` 
Which Is the right way to launch the container with different port?

Thanks.

- `docker-compose.yml`:
```
  opencart3_db:
      container_name: opencart3_db
      image: mariadb
      environment:
         MYSQL_ROOT_PASSWORD: opencart3_psw
         MYSQL_DATABASE: opencart3_db
         MYSQL_USER: opencart3_db_usr
         MYSQL_PASSWORD: opencart3_db_psw
      ports:
         - 3311:3306

   opencart3:
    image: 'bitnami/opencart:3'
    environment:
      MARIADB_HOST: opencart3_db
      MARIADB_PORT_NUMBER: 3306
      OPENCART_DATABASE_USER: opencart3_db_usr
      OPENCART_DATABASE_PASSWORD: opencart3_db_psw
      OPENCART_DATABASE_NAME: opencart3_db
      OPENCART_HOST: localhost:8084
    ports:
      - 8084:80
    depends_on:
      - opencart3_db
```
**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea
 Built:             Wed Nov 13 07:22:34 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea
  Built:            Wed Nov 13 07:29:19 2019
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker info`:

```
Client:
 Debug Mode: false

Server:
 Containers: 7
  Running: 1
  Paused: 0
  Stopped: 6
 Images: 15
 Server Version: 19.03.5
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: b34a5c8af56e510852c35414db4c1f4fa6172339
 runc version: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 4.9.184-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 1.952GiB
 Name: docker-desktop
 ID: IGBV:VMHR:4G5F:YDGF:MTZG:BTGV:R3G4:ZIBC:TRCI:HFBL:BT4E:Q6LE
 Docker Root Dir: /var/lib/docker
 Debug Mode: true
  File Descriptors: 39
  Goroutines: 57
  System Time: 2019-12-03T14:25:48.3975466Z
  EventsListeners: 2
 HTTP Proxy: gateway.docker.internal:3128
 HTTPS Proxy: gateway.docker.internal:3129
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: true
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.21.2, build a133471
docker-py version: 3.5.1
CPython version: 2.7.16
OpenSSL version: LibreSSL 2.2.7
```

**Comment by tompizmor** [comment]
Hi @mattia85,

I think currently is not possible to change the default port. Obviously, you can change the port mapping in the docker-compose but then images stop showing on the page. That is why you wanted to specify the port right?

I create a task to properly check the issue but I can't give you any ETA on when will be done. 
Thanks for reporting the issue!

**Comment by overclockedllama** [comment]
have you considered using a reverse proxy @mattia85

**Comment by Fuseteam** [comment]
this should be solved with the solution from #66
aka change the apache ports and map your port to that port like 8084:8084

**Comment by andresbono** [comment]
Thanks for sharing it here, @Fuseteam!
